doc/source/user_guide/io.rst

```diff
@@ -2986,7 +3003,7 @@ Read a URL with no options:
 Read in the content of the "books.xml" file and pass it to ``read_xml``
 as a string:
 
-.. ipython:: python
+.. ipython:: python :okexcept:
```

Comment:
Why was this added?

Reply:
Hi Matthew, thanks for your help. I added this initially because I was having errors when making, but I will remove this and repush